packages/commonwealth/client/scripts/views/pages/CreateCommunity/utils.ts

```diff
@@ -1,15 +1,18 @@
+import React from 'react';
+
 import { featureFlags } from 'helpers/feature-flags';
 import { CWFormStepsProps } from 'views/components/component_kit/new_designs/CWFormSteps/CWFormSteps';
 
 export enum CreateCommunityStep {
-  CommunityTypeSelection = 0,
-  BasicInformation = 1,
-  CommunityStake = featureFlags.communityStake ? 2 : undefined,
-  Success = featureFlags.communityStake ? 3 : 2,
+  CommunityTypeSelection = 'CommunityTypeSelection',
```

Comment:
If it helps, maybe we could append the step number at the start, like `1_CommunityTypeSelection`, but that's very minor. Plz resolve this if this is unnecessary

Reply:
I think I will keep it as it is. At first I used enum but when I realised that steps are dynamic depending on selected chain in form step, I had to changed it. I think it works fine now.